WordPressAuthenticator/Signin/LoginPrologueViewController.swift

```diff
@@ -55,6 +55,9 @@ class LoginPrologueViewController: LoginViewController {
         }
 
         defaultButtonViewMargin = buttonViewLeadingConstraint?.constant ?? 0
+        if let backgroundImage = WordPressAuthenticator.shared.unifiedStyle?.prologueBackgroundImage {
```

Comment:
👋 Hey @Gio2018, would it make sense for this to live in the `styleBackground` function?